apps/stripe/src/modules/app-config/stripe-config.ts

```diff
@@ -0,0 +1,63 @@
+import { err, ok } from "neverthrow";
+
+import { StripePublishableKey } from "@/modules/stripe/stripe-publishable-key";
+import { StripeRestrictedKey } from "@/modules/stripe/stripe-restricted-key";
+
+export class StripeConfig {
+  private constructor(
+    private deps: {
+      name: string;
+      id: string;
+      restrictedKey: StripeRestrictedKey;
+      publishableKey: StripePublishableKey;
+    },
```

Comment:
these are not deps, these are fields, sometimes called props. these are not dependencies

why don't you just put these as class members?